src/Fable.Import.Electron.fs

```diff
@@ -22,24 +22,33 @@ module ElectronPrivate =
 
 module Electron =
     type [<Import("EventEmitter", "electron")>] EventEmitter() =
-        interface NodeJS.EventEmitter with
-            member __.addListener(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.on(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.once(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.removeListener(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.removeAllListeners(?``event``: string): NodeJS.EventEmitter = failwith "JS only"
-            member __.setMaxListeners(n: int): unit = failwith "JS only"
+        interface Node.Events.event_types.EventEmitter with
+            member __.addListener(``event``: U2<string, Symbol>, listener: Function): Node.Events.event_types.EventEmitter = failwith "JS only"
+            member __.on(``event``: string, listener: ('a -> unit)): Node.Events.event_types.EventEmitter = failwith "JS only"
+            member __.on(``event``: Symbol, listener: ('a -> unit)): Node.Events.event_types.EventEmitter = failwith "JS only"
+            member __.once(``event``: string, listener: ('a -> unit)): Node.Events.event_types.EventEmitter = failwith "JS only"
+            member __.once(``event``: Symbol, listener: ('a -> unit)): Node.Events.event_types.EventEmitter = failwith "JS only"
+            member __.removeListener(``event``: U2<string, Symbol>, listener: Function): Node.Events.event_types.EventEmitter = failwith "JS only"
+            // member __.removeAllListeners(?``event``: U2<string, Symbol>): Node.Events.event_types.EventEmitter = failwith "JS only"
+            member __.setMaxListeners(n: int): Node.Events.event_types.EventEmitter = failwith "JS only"
             member __.getMaxListeners(): int = failwith "JS only"
-            member __.listeners(``event``: string): ResizeArray<Function> = failwith "JS only"
-            member __.listenerCount(``type``: string): int = failwith "JS only"
+            member __.listeners(``event``: U2<string, Symbol>): ResizeArray<Function> = failwith "JS only"
```

Comment:
I think @jgrund we can just use interfaces instead of fake classes ;) But that's ok, I'll merge the PR and you can do the fixes later. Thanks a lot for your work @mizzle-mo!